### ISSUE TITLE
menu will not open

### ISSUE THREAD

**Issue by Fr0s7yF3l1n3** [WildfireRomeo/WildfireFemaleGenderMod#104]
Despite making sure the menu was still linked with the G key, it cannot be opened. Please help I just wanna be cute

**Comment by Scraft161** [comment]
just to prevent dumb things from taking days to solve, have you made sure it's the only thing bound to `G`?
this includes keybindings in the vanilla menu, but also ones registered by other mods you have installed that may have their own menu or keybinding system.

**Comment by Fr0s7yF3l1n3** [comment]
I went through the entire keybind section to make sure it was the only one bound to G, yes

**Comment by Scraft161** [comment]
Can you answer the questions below then?

- Do you use forge or fabric to load the mod?
- Which version of the mod are you using?
- What other mods (if any) do you have installed?
- Can you send me the minecraft log (you can get the one from the last time you started the game from `.minecraft/logs/latest.log`

**Comment by Fr0s7yF3l1n3** [comment]
Ive got Fabric, 1.19.  I didnt get any results despite searching from the link you provided.. 
There are several mods that Im certain Would effect it had I had them when I first started, so the list may not be helpful as well. This mod was one of the first I added and it didnt work then as well and truthfully Im not sure Ill use it now that I have MCA. So unless this may help someone else I may close the discussion. Apologies if Ive wasted your time
